### DISCUSSION TITLE
AlphaViz

### DISCUSSION THREAD

**Discussion by singjc** [MannLabs/alphatims#159] [General]
Hello,

I had a question about post-data analysis visualization.

I forked a branch of alphatims to implement visualization of already analyzed data (specifically OpenSwathWorkflow), but I see that there is another software `AlphaViz` that is used for the purpose of visualizing analyzed data. I can't seem to find `AlphaViz`, is this still a in a private repo or somewhere else?

Best,

Justin

**Comment by EugeniaVoytik** [comment]
Hello Justin,

Thank you very much for your question and your interest in our software tools. 🙏🏻 Yes, you are right, we are currently developing the AlphaViz tool, an automated visualisation pipeline that links the identifications found by different software tools such as AlphaPept, MaxQuant or DIA-NN with the original Bruker or Thermo raw data. This will make it easy to assess the individual quality of these identifications or the overall quality of the samples. In the first release, scheduled before Christmas, we plan to include only the tools mentioned, but after the release the well-documented code will be available for everyone. If you are interested in the OpenSwathWorkflow, we look forward to any contributions! 😊

Best,
Jane

**Reply by singjc** [reply]
Dear Jane,

Thank you for your reply! That's great, I look forward to its public release :)

Best,

Justin

**Comment by mschwoer** [comment]
see https://github.com/MannLabs/alphaviz :-)
